bananagram-frontend/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM node:10.16.0-jessie as build-stage
+WORKDIR /app
+COPY package*.json /app/
```

Comment:
Step 3/8 : COPY package*.json /app/

shoul ? look for yarn.lock

Reply:
Resolved in e4150c6